jwala-persistence/src/test/java/com/cerner/jwala/persistence/jpa/domain/constraint/validator/PathValidatorTest.java

```diff
@@ -54,9 +54,11 @@ public void testIsValid() throws Exception {
         assertEquals("invalid dirAndFilename", constraintViolations.iterator().next().getMessage());
         assertTrue(constraintViolations.size() == 1);
 
-        constraintViolations = validator.validate(new PathsWrapper("c::\\test/any", "c:\\jdk.zip", existingFile));
-//        assertEquals("invalid dir", constraintViolations.iterator().next().getMessage());
-        assertTrue(constraintViolations.size() == 0);
+        if (SystemUtils.IS_OS_WINDOWS) {
```

Comment:
Looks good! But I need to test this in windows before I can approve. Give me a sec...